docs/guide/episodes/episode_01.md

```diff
@@ -0,0 +1,110 @@
+# 1. Metaflow basics - fetching auxillary data from the web
+
+:warning: This is not a Metaflow tutorial! It is assumed that you have already looked through the [official Metaflow tutorials](https://docs.metaflow.org/getting-started/tutorials) and familiarised yourself with the [Metaflow docs](https://docs.metaflow.org/).
+
+## Standard Industrial Taxonomy (SIC)
+
+The first dataset that is fetched and processed is an Excel spreadsheet containing the codes and names of each of the 5 levels of the Standard Industrial Classification (SIC) taxonomy.
+Later in the project when we come to analyse whether industry has an effect on the performance of a business' website this will be done using the SIC taxonomy - it's important that we are able to convert the (zero-padded) numeric codes into their description names when interpreting and reporting results.
+
+The flow code can found in [`kuebiko/pipeline/sic/flow.py`\*](../pycco/kuebiko/pipeline/sic/flow.html) and its utility functions (which are not a focus of this episode) in [`kuebiko/pipeline/sic/utils.py`](../pycco/kuebiko/pipeline/sic/utils.html).
+
+Lets recap a few Metaflow commands:
+
+-   `python kuebiko/pipeline/sic/flow.py show` will give us the docstrings and basic structure of the flow.
+
+-   `python kuebiko/pipeline/sic/flow.py run` will run the flow.
+
+-   `python kuebiko/pipeline/sic/flow.py dump <run id>/end` will dump a summary of the artifacts for a given run ID.
+
+!!! info "Metaflow defaults"
+
+    `.envrc` specifies:
+
+    - `METAFLOW_DEFAULT_DATASTORE=local`
+    - `METAFLOW_PROFILE=ds-cookiecutter`
+
+    [`direnv`](https://direnv.net/) automatically loads these environment variables (after you run `direnv allow` - a security mechanism to avoid changes to `.envrc` automatically executing unsafe code).
+
+    The end effect is that Metaflow will use the configuration in `~/.metaflowconfig/config_ds-cookiecutter.json` that `ds-cookiecutter` sets up, and then override the default datastore to be local (data artifacts are persisted on your local machine in a `.metaflow/` folder not to S3).
```

Comment:
@bishax One thing I'm not clear about: Are artifacts that have been created in two different runs but are in fact identical copies (eg, the same input csv table) stored somewhat cleverly (eg, in the style of git versioning, keeping track of differences) - or are they kept as two copies and hence take up 2x disk space?

Reply:
Related to this, I'm wondering what's the best practice when developing/prototyping flows? In the open jobs project I ran into the scenario that I'm running the flows many times until I'm happy and don't have bugs. As a result, there are a lot of persisted runs in the metaflow folder (and most of it is trash). Is that something to worry about and how best to deal with it?